raiden/tests/integration/test_integration_pfs.py

```diff
@@ -65,9 +65,6 @@ def wait_all_apps(raiden_network: List[RaidenService]) -> None:
 @raise_on_failure
 @pytest.mark.parametrize("number_of_nodes", [3])
 @pytest.mark.parametrize("channels_per_node", [0])
-@pytest.mark.parametrize(
-    "broadcast_rooms", [[DISCOVERY_DEFAULT_ROOM, PATH_FINDING_BROADCASTING_ROOM]]
-)
```

Comment:
Why is the parametrization removed here, but not in the tests below?

Reply:
`broadcast_rooms = [DISCOVERY_DEFAULT_ROOM]` is the default from the `conftest.py` fixtures and the services rooms are factored out - this is why it was removed here.
The parametrisation in the other tests makes sure the default fixture is overwritten and excludes the discovery room from the broadcast rooms. Since there are no other rooms anymore, the list is empty.
I guess the "lack of discovery room" is not strictly necessary for the test, but it is how it was implemented before.